tools/cockpit.spec

```diff
@@ -459,7 +459,7 @@ Requires(post): (policycoreutils if selinux-policy-%{selinuxtype})
 Conflicts: firewalld < 0.6.0-1
 Recommends: sscg >= 2.3
 Recommends: system-logos
-Suggests: sssd-dbus
+Suggests: sssd-dbus >= 2.6.2
```

Comment:
This is not a typo -- 2.6.2 fixes [this bug](https://github.com/SSSD/sssd/issues/5911). But if it seems weird, I'm also happy to set this to 2.6.1 (same for the spec file). It doesn't matter much anyway.